app/integrations/discord/interactions/resolvers/message.rb

```diff
@@ -11,10 +11,12 @@ class Message < DiscordEngine::Resolvers::Resolver
         def execute_action
           message = create_message
           message_component = create_message_component(message)
-          send_message(message_component)
+          response = send_message(message_component)
+          link_discord_message(message, response)
 
           @content = '✅ Message created!'
-        rescue Messages::CreationFailed => e
+        rescue Messages::CreationFailed, RuntimeError => e
```

Comment:
I dont really like RuntimeError, its generic, and i'm using it for responsing to a Discordrb::API errors